packages/fluentui/react-builder/src/components/ComponentTree.tsx

```diff
@@ -78,7 +78,6 @@ const jsonTreeToTreeItems: (
     onTitleClick: handleSelectedComponent,
     id: tree.uuid as string,
     title: {
-      children: titleRenderer,
```

Comment:
Is this intentional?

Reply:
It's a fix from Lukas to make the component tree accessible again.

Reply:
So I guess we also need to remove it from the props.

Reply:
Okay, the change has been merged to the upstream, so nothing to change in this PR>